ansible_collections/arista/cvp/docs/how-to/cvp-authentication.md

```diff
@@ -28,15 +30,28 @@ ansible_httpapi_use_ssl: true
 ansible_httpapi_validate_certs: false
 ```
 
+Alternatively __user token__ can be used just as with CVaaS. See [How to generate service account tokens](#how-to-generate-service-account-tokens) for the token generation steps.
+
+```yaml
+# Default Ansible variables for authentication
+ansible_host: < IP address or hostname to target >
+ansible_user: svc_account # Set this username to user api_token in cvprac
```

Comment:
```suggestion
# The username "svc_account" will change authentication to use api_token towards CVP.
# The username does not need to match anything defined on CVP, since the token 
# contains all the required information.
ansible_user: "svc_account" 
```